dex/overlays/nerc-ocp-infra/configmaps/files/config.yaml

```diff
@@ -22,6 +22,11 @@ staticClients:
       - https://vault-ui-vault.apps.nerc-ocp-infra.rc.fas.harvard.edu/oidc/callback
       - https://vault-ui-vault.apps.nerc-ocp-infra.rc.fas.harvard.edu/ui/vault/auth/oidc/oidc/callback
     secretEnv: VAULT_SECRET
+  - id: grafana
+    name: Grafana
+    redirectURIs:
+      - https://logging-grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu/login/generic_oauth
+    secretEnv: GRAFANA_SECRET
```

Comment:
Is GRAFANA_SECRET and VAULT_SECRET made available through this: [https://github.com/OCP-on-NERC/nerc-ocp-config/blob/6b360c27f91be4dbfff1984ce55920b504f645bf/dex/base/deployments/dex.yaml#L42](https://github.com/OCP-on-NERC/nerc-ocp-config/blob/6b360c27f91be4dbfff1984ce55920b504f645bf/dex/base/deployments/dex.yaml#L42) in the dex deployment?

Reply:
@dystewart yes, that looks like where it references the dex-clients
```yaml
          envFrom:
            - secretRef:
                name: dex-clients
```